pg_search/src/index/reader/index.rs

```diff
@@ -360,6 +382,8 @@ impl SearchIndexReader {
                 &self.searcher,
```

Comment:
This method does not need to be public, because it's only used inside `SearchIndexReader`.

And ... it looks like `SearchIndexReader` now already _has_ a `Query` field. So ... this method could maybe be removed?